tests/unittests/test_exception_handling.py

```diff
@@ -1,38 +1,10 @@
 import unittest
 from unittest import mock
+from parameterized import parameterized
 
 import requests
-import tap_typeform.http as client_
-
-
-
-def mocked_session(*args, **kwargs):
-    class Mocksession:
-        def __init__(self, json_data, status_code, content, headers, raise_error):
-            self.text = json_data
-            self.status_code = status_code
-            self.raise_error = raise_error
-            if headers:
-                self.headers = headers
-
-        def raise_for_status(self):
-            if not self.raise_error:
-                return self.status_code
-
-            raise requests.HTTPError("sample message")
-
-        def json(self):
-            return self.text
-
-    arguments_to_session = args[0]
-
-    json_data = arguments_to_session[0]
-    status_code = arguments_to_session[1]
-    content = arguments_to_session[2]
-    headers = arguments_to_session[3]
-    raise_error = arguments_to_session[4]
-    return Mocksession(json_data, status_code, content, headers, raise_error)
-
+from tap_typeform.client import ERROR_CODE_EXCEPTION_MAPPING
+import tap_typeform.client as client_
 
```

Comment:
unit tests for the TypeformError for method `raise_for_error` are missing.

Reply:
Added unittests